<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgNetCamera</class>
 <widget class="QDialog" name="DlgNetCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络摄像头</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DlgNetCamera {
	background: rgb(255,255,255);
}
QComboBox {
	background: transparent;
	border: 1px solid rgb(221,221,221);
	border-radius: 4px;
	min-height: 30px;
	padding-left: 8px;
	padding-right: 8px;
}
QComboBox:hover {
	border-color:rgb(57,129,252);
}
QComboBox::drop-down {
	padding-top: 5px;
	image: url(:/Resources/dropDown.png);
}
QComboBox QAbstractItemView{
	margin-top:1;
	outline:0;	
	border:1px solid rgb(225,225,225);
	background-color:rgb(255,255,255);
	selection-background-color:rgb(80,145,255);
}
QComboBox QAbstractItemView::item{
	min-height:30px;
	max-height:30px;
}
QComboBox QAbstractItemView::item:hover{
    background-color:rgb(240,240,240);
	color:rgb(45,45,45);
}
QComboBox QAbstractItemView::item:selected{
    background-color:rgb(240,240,240);
	color:rgb(45,45,45);
}
QSlider {
	min-height: 20px;
	max-height: 20px;
	padding-left: 8px;
	padding-right: 8px;
}
QSlider::add-page:horizontal {       
	background-color:rgb(223,223,223);  
}  
QSlider::sub-page:horizontal {  
	background-color: rgb(57,129,252);
}  
QSlider::groove:horizontal {  
	background:transparent;  
	height:2px;
}  
QSlider::handle:horizontal {
	height: 12px;
	width: 12px;
	margin-top: -5px;
	margin-bottom: -5px;
	border-radius: 6px;
	background: rgb(57,129,252);
}  </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="horizontalWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="editUrl">
        <property name="placeholderText">
         <string>请输入rtmp、rtsp地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOperate">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="videoLayout"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
